package.json

```diff
@@ -44,6 +44,6 @@
   },
   "dependencies": {
     "dotenv": "^16.0.3",
-    "meilisearch": "^0.31.1"
+    "meilisearch": "0.32.2"
```

Comment:
I know it technically doesn't change anything and at the end of the day on apps you chose where you want to put your dependencies, but out of curiosity, why is one in `devDependencies` and the other in `dependencies` ?

Reply:
I build the Nuxt 3 app as a static site. So I see everything used to build the Nuxt app as a dev dependencies: they get bundle via Vite but are not installed in the production site.

However, should you want to run the `setup` script, you really need to install meilisearch and dotenv. In a way, these deps are required to manage the production app.

I don't have the strongest opinions on this tbh. I find deps vs devDeps to become more and more vague with the use of build tools.